index.html

```diff
@@ -196,6 +196,68 @@ <h3>Multi-Post Stories<br />Gain+Glory</h3>
           </div>
         </div>
       </div>
+      <div id="mobile-popup-container" class="display-none">
+        <div id="mobile-popup">
+          <div id="mobile-popup-image-div">
+            <img src="img/MobilePopup.jpg" alt="">
+            <button class="mobile-popup-close-button">&times;</button>
+          </div>          
+          <h2>
+            Keeping track of hundreds of
+            components
+          </h2>
+          <div class="card-lang">
+            <ul class="card-list d-flex">
+              <li class="card-list-element">Ruby on Rails</li>
+              <li class="card-list-element">CSS</li>
+              <li class="card-list-element">JavaScript</li>
+            </ul>
+          </div>
+          <p>
+            Lorem Ipsum is simply dummy text of the printing and typesetting industry. 
+            Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, 
+            when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem 
+            Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's 
+            standard dummy text ever since the 1500s, 
+            when an unknown printer took a galley of type and scrambled it 1960s with the relea
+          </p>
+          <div id="mobile-popup-buttons" class="d-flex">
+            <a href="#" class="pop-up-button g-btn" id="see-live">See Live <img class="popicons" src="img/PopupLive.png" alt=""></a>
+            <a href="#" class="pop-up-button g-btn" id="see-source">See Source <img class="popicons" src="img/PopupGit.png" alt=""></a>
```

Comment:
Please add links for the live version and source code.

Reply:
This issue is well resolved! ✔️